wherehows-frontend/app/controllers/Application.java

```diff
@@ -18,7 +18,8 @@
 import dao.FlowsDAO;
 import dao.MetricsDAO;
 import dao.UserDAO;
-import java.io.InputStream;
+
+import java.io.*;
```

Comment:
Is it possible to not use * and expend the list if it's only for a couple imports.